src/main/java/org/solid/testharness/http/AuthManager.java

```diff
@@ -17,13 +19,21 @@
 public class AuthManager {
     private static final Logger logger = LoggerFactory.getLogger(AuthManager.class);
 
-    private ObjectMapper objectMapper = new ObjectMapper();
+    @Inject
+    UserCredentials aliceCredentials;
+
+    @ConfigPrefix("bob")
+    UserCredentials bobCredentials;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Get this via CDI Injection

Reply:
I switched in quarkus-jackson in place of com.fasterxml.jackson.core and was able to use CDI. Thanks